src/test/java/nextstep/courses/domain/enums/ApprovalStateTest.java

```diff
@@ -0,0 +1,27 @@
+package nextstep.courses.domain.enums;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class ApprovalStateTest {
+
+    @Test
+    void 동일비교 () {
+        assertThat(ApprovalState.APPROVED).isEqualTo(ApprovalState.APPROVED);
```

Comment:
```suggestion
        assertThat(ApprovalState.APPROVED  == ApprovalState.APPROVED).isTrue();
```
enum은 jvm 상에서 싱글 인스턴스를 보장하기 때문에 == 비교 가능하다.
굳이 이와 같은 단위 테스트는 필요 없지 않을까?